src/NuGet/EtwCollector/ApplicationInsights.config.install.xdt

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0"?>
+<ApplicationInsights xmlns="http://schemas.microsoft.com/ApplicationInsights/2013/Settings" xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
+  <TelemetryModules xdt:Transform="InsertIfMissing">
+    <Add Type="Microsoft.ApplicationInsights.EtwCollector.EtwTelemetryModule, Microsoft.ApplicationInsights.EtwCollector" xdt:Locator="Match(Type)" xdt:Transform="InsertIfMissing">
+      <Sources>
+        <!-- Replace/add your own ETW provider as necessary. -->
+        <Add ProviderName="Some-ETW-Provider-Name" Level="Warning"/>
```

Comment:
so you want a "broken" provider be registered by default? Can you include some useful provider instead? Like "process start/process stop" provider. Or have a hardcoded logic that will not try to register this provider